auth-server/test/resolvers/mutation/addressLinkConfirm.spec.ts

```diff
@@ -11,34 +11,28 @@ import Address from '../../../src/model/Address';
 import User from '../../../src/model/User';
 import addressLinkConfirm from '../../../src/resolvers/mutation/addressLinkConfirm';
 import addressLinkStart from '../../../src/resolvers/mutation/addressLinkStart';
-import signup from '../../../src/resolvers/mutation/signup';
 import { Context, NetworkEnum } from '../../../src/types';
 import messages from '../../../src/utils/messages';
+import { getNewUserCtx } from '../../helpers';
 
 describe('addressLinkConfirm mutation', () => {
-	let signupResult: any;
-	const fakectx: Context = {
-		req: {
-			headers: {}
-		},
-		res: {
-			cookie: () => {}
-		}
-	} as any;
+	let signupUserId = 0;
+	let fakectx: Context;
 	const email = 'test@email.com';
 	const password = 'testpass';
 	const username = 'testuser';
 	const name = 'test name';
 
 	before(async () => {
-		signupResult = await signup(undefined, { email, password, username, name }, fakectx);
-		fakectx.req.headers.authorization = `Bearer ${signupResult.token}` // eslint-disable-line
+		const result = await getNewUserCtx(email, password, username, name);
```

Comment:
typescript complaining 

Left side of comma operator is unused and has no side effects.ts(2695)
If i do 
```
let { ctx: fakectx, userId: signupUserId } = await getNewUserCtx(email, password, username, name);
```
It will be local to before only

Reply:
ok then let's forget about it, not a big deal.